_includes/pbc/all/install-features/202304.0/install-glue-api/install-the-picking-app-glue-api.md

```diff
@@ -0,0 +1,197 @@
+
+
+
+This document describes how to integrate the Picking App feature API into a Spryker project.
+
+## Install feature core
+
+Follow the steps below to install the Picking App feature API.
+
+## Prerequisites
+
+To start feature integration, integrate the required features and Glue APIs:
+
+| NAME             | VERSION          | INTEGRATION GUIDE                                                                                                                                                |
+|------------------|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Spryker Core API | {{site.version}} | [Glue API: Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/glue-api/glue-api-spryker-core-feature-integration.html) |
+| Picking App      | {{site.version}} | [Picking App feature integration](/docs/scos/dev/feature-integration-guides/{{site.version}}/picking-app-feature-integration-guide.md)                           |
+
+## 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer install spryker/user-backend-api:"^0.1.0" spryker/warehouse-user-backend-api:"^0.1.0" --update-with-dependencies
```

Comment:
Let's move it to main IG, we try to follow API first concept, it means that API modules now part of main feature.

Reply:
I agree with this.